src/vmm/src/logger/metrics.rs

```diff
@@ -78,6 +78,26 @@ use crate::devices::virtio::vhost_user_metrics;
 use crate::devices::virtio::virtio_block::metrics as block_metrics;
 use crate::devices::virtio::vsock::metrics as vsock_metrics;
 
+#[macro_export]
```

Comment:
`#[macro_export]` exports macro for the whole crate. Instead it would be better to use `pub(crate) use <macro_name>`. This way macro will be a part of the module and can be imported using normal `used mod::macro` syntax.

Reply:
removed the macro and updated code as per other comments.